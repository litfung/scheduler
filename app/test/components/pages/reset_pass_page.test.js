import React from 'react';
import { shallow } from 'enzyme';
import { Provider } from 'react-redux';

import ResetPasswordPage from '../../../src/client/components/pages/reset_pass_page';
import configureStore from '../../../src/helpers/configure_store';

test('Snapshot for ResetPasswordPage Component', () => {
  const wrapper = shallow(
    <Provider store={configureStore({ get: jest.fn(() => {}) })}>
      <ResetPasswordPage.component
        handleAppMode={jest.fn(() =>  {})}
      />
    </Provider>
  );
  expect(wrapper).toMatchSnapshot();
});
