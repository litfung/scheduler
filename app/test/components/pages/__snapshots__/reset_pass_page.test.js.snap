// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot for ResetPasswordPage Component 1`] = `
<Connect(ResetPasswordPage)
  handleAppMode={[MockFunction]}
/>
`;
